spec/acceptance/shared_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'spec_helper_acceptance'
+
+describe 'Simple installation' do
+  interface = 'eth0'
+  config_file = fact('os.family') == 'Archlinux' ? '/etc/dhcpd.conf' : '/etc/dhcp/dhcpd.conf'
+
+  it_behaves_like 'an idempotent resource' do
+    let(:manifest) do
+      <<-EOS
+      $interface = $facts['networking']['interfaces']['#{interface}']
+
+      class { 'dhcp':
+        interfaces     => ['#{interface}'],
```

Comment:
this variable interpolation looks a bit complicated. can't we just do:
```suggestion
        interfaces     => [$facts['networking']['interfaces']['eth0']],
```
Then you can drop line 10

Reply:
I've copied the test from the repo, it use the variable interface during the test
I can replace it in all the manifest with the long facts

Reply:
No, the current code is correct.